dcs/planes.py

```diff
@@ -381,7 +381,6 @@ class Pylon9:
 
 class F_A_18C(PlaneType):
     id = "F/A-18C"
-    flyable = True
```

Comment:
should this be removed?

Reply:
> should this be removed?

I'm pretty sure it should, it's the regular C-variant which is AI only afaik. The flyable F-18's ID should be "FA-18C_hornet".